scripts/remote-tmux-reconnect-auth-harness.sh

```
@@ -0,0 +1,379 @@
+fi
+
+log "=== scenario 3: completing the login resumes the mirror"
+restore_auth
+# Stand in for the user finishing the login: open the shared master at cmux's own
+# ControlPath, which is what the login pane's `ssh` invocation does. cmux probes its own
+# socket, so a master opened anywhere else would be invisible to it.
+master_open_attempt() {
+  ssh -o ControlMaster=auto -o ControlPath="$CMUX_CP" \
+      -o ControlPersist=2m -n -T "$HOST" true 2>/tmp/harness-master-open.err
+  ssh -O check -o ControlPath="$CMUX_CP" "$HOST" >/dev/null 2>&1
+}
```

Shellcheck (0.11.0)

[info] This function is never invoked. Check usage (or ignored if invoked indirectly).

(SC2329)